src/main/java/com/marklogic/client/ext/file/JarDocumentFileReader.java

```diff
@@ -96,7 +96,7 @@ public void setUriPrefix(String uriPrefix) {
 				Resource[] r = resolver.getResources(finalPath);
 				list.addAll(Arrays.asList(r));
 			} catch (IOException e) {
-				e.printStackTrace();
+				throw new RuntimeException("Unable to find resources at path: " + path, e);
```

Comment:
@rjrudin Let me get back to you on this one. I may have printed because there were legit cases where you don't want to throw